static/js/util/sorting.js

```diff
@@ -1,10 +1,12 @@
 // @flow
```

Comment:
doesn't seem like that name applies anymore. `sortEntriesByDate` or something would be more on the mark, i think